### ISSUE TITLE
.ent: can "export appearance with rig" button call the WScript?

### ISSUE THREAD

**Issue by manavortex** [WolvenKit/WolvenKit#1422]
<img width="433" alt="image" src="https://github.com/WolvenKit/WolvenKit/assets/965785/3b9725e1-4578-4150-b613-66115cd4a29f">

I think it'd be cool if the button in the .ent preview would automatically call the correct exporting WScript. We have them, they work well, why not hook them up to the GUI? (Genuine question)

**Comment by Simarilius-uk** [comment]
The script will need modifying so it can be called with an ent passed to it, as the preview doesnt mean its in the project. also means your adding a lot of stuff to the project not just exporting.

**Comment by manavortex** [comment]
Could make that transparent to the user somehow, though? Or maybe have two buttons?

**Comment by Simarilius-uk** [comment]
Yeah, just flagging that the current behaviour and the script are very different. The current button (when it works) chucks out a single glb with everything rolled into it, thats totally independent of the project. The script chucks loads of things into the project and exports them. Two buttons would probably be better.
